src/utils/imageLargeView/imageLargeView.html

```diff
@@ -0,0 +1,6 @@
+<div class="c-image-large">
+    <div class="c-image-large__header l-browse-bar">
+        <span class="c-object-label__name">{{ time }}</span>
+    </div>
+    <img :src="imageSrc" alt="image" :style="filter" />
```

Comment:
Just saw alt="image", don't think it's a blocker, but might be nice to pass in something more descriptive if we have it, but maybe we wouldn't know what would be consistently provided that could be used. Maybe the domainObject.name if that makes sense?

Reply:
hmm, thinking of passing that name in meta, as this can be used by anyone and may not have domainObject.